### PR TITLE
Show board in Webapp

### PR DESCRIPTION
**Pull request by adguernier**
[Trello Card](https://trello.com/c/ghjVGpXv/20-1-as-hayao-i-want-to-see-a-real-hexagon-board-game)

[Capture vidéo du 25-04-2023 12:06:26.webm](https://user-images.githubusercontent.com/7949510/234244836-f1a1d38f-4a28-4e2c-b8e7-558a04b83acc.webm)

### CODE REVIEW

**Comment by fzaninotto** [comment]
I don't know what this is about

File: webapp/src/app.controller.spec.ts [hunk 1]
```diff
@@ -14,9 +14,9 @@ describe('AppController', () => {
     appController = app.get<AppController>(AppController);
   });
 
-  describe('root', () => {
+  describe("root", () => {
```
Comment (paulSanchezMarmelab) [inline webapp/src/app.controller.spec.ts:17]: This test isn't valid.

**Comment by paulSanchezMarmelab** [comment]
For later, maybe `makefile.ci.yml` should start both `make test-cli` and `make test-webapp`

File: webapp/public/app.css [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  background: #112b3c;
+  --s: 100px;
+  --m: 4px;
```
Comment (fzaninotto) [inline webapp/public/app.css:24]: Could you choose more explicit variable names?

File: webapp/public/app.css [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+  height: 900px;
+  font-size: 0;
+}
+
+ul {
+  list-style: none;
+  display: flex;
+  gap: 2px;
+  position: relative;
+  top: -20px;
```
Comment (fzaninotto) [inline webapp/public/app.css:49]: It seems a bit too specific for such a generic element. Shouldn't you style uls of a particular class instead?

File: webapp/public/app.css [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+}
+
+.tile {
+  width: var(--s);
+  margin: var(--m);
+  height: calc(var(--s) * 1.1547);
```
Comment (fzaninotto) [inline webapp/public/app.css:55]: Magic number, please make it a const with an expressive name, and/or link to the computation for it

File: webapp/public/app.css [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+  font-size: 2.5rem;
+  color: white;
+  margin-bottom: calc(var(--m) - var(--s)*0.2885);
```
Comment (fzaninotto) [inline webapp/public/app.css:58]: same

File: webapp/src/app.controller.ts [hunk 6]
```diff
@@ -1,12 +1,218 @@
...
 @Controller()
 export class AppController {
-  constructor(private readonly appService: AppService) {}
-
   @Get()
-  getHello(): string {
-    return this.appService.getHello();
+  @Render("index")
+  root() {
+    const game = [
```
Comment (fzaninotto) [inline webapp/src/app.controller.ts:7]: :+1: To work on the UI, you use static data. Great shortcut!

File: webapp/views/index.hbs [hunk 7]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>App</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
```
Comment (fzaninotto) [inline webapp/views/index.hbs:6]: please import static assets, for the sake of learning how to serve them ;)

**Review by fzaninotto — CHANGES_REQUESTED** [review]
Great UI! I just think that the played locations deserve a better UI.

**Review by Nitix — APPROVED** [review]

**Pull request merged**